### PR TITLE
Add and remove books feature

### PR DESCRIPTION
**Pull request by micheaol**
### In this pull request:

- Import create the store
- Import combineReducers - We will only write a reducer for 'books' in this step, but in the future, we might want to add 'categories' or other reducers as well.
- Import applyMiddleware - To use some handy Redux middleware, like a redux logger. It's optional, but it will help you a lot in the development process. To install it, you'll need to run:
- Import your books reduced, which you are going to write in /src/redux/books directory.
- Write your book's actions and reducer (in /src/redux/books/books.js)
- Use the redux-logger to check if your application is working correctly - Open your browser's console and look for the logger 
 messages. Upon every action dispatch, the logger will print:
      - prev state - the state of the whole Redux store before the current action was dispatched.
      - action - the action object that is being dispatched.
      - next state - the state of the Redux store after the current action was dispatched.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @micheaol 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (mikenath223) [inline src/redux/configureStore.js:1]: - [ ] Remove this eslint disable line and fix the issue in the code.

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-useless-path-segments */
```
Comment (mikenath223) [inline src/index.js]: - [ ] Remove this eslint disable line and fix the issue in the code.

File: src/components/BookList/index.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 /* eslint-disable no-unused-vars */
```
Comment (mikenath223) [inline src/components/BookList/index.js]: - [ ] Remove this eslint disable line and fix the issue in the code.

File: src/components/AddBook/index.js [hunk 4]
```diff
@@ -1,22 +1,45 @@
+/* eslint-disable no-unused-vars */
```
Comment (mikenath223) [inline src/components/AddBook/index.js]: - [ ] Remove this eslint disable line and fix the issue in the code.

File: package.json [hunk 5]
```diff
@@ -52,6 +54,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "redux": "^4.1.2",
```
Comment (mikenath223) [inline package.json:57]: - [ ] Also update the readme.md file following [this template](https://github.com/microverseinc/readme-template).

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @micheaol,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**